### DISCUSSION TITLE
Heic to JPG

### DISCUSSION THREAD

**Discussion by GingerNinjaa** [dlemstra/Magick.NET#1082] [Help]
Hi,

I have issue with conversion between .heic to jpg. Can someone send me a sample of this conversion?

**Comment by dlemstra** [comment]
Can you explain in more detail what kind of issue you are experiencing?

**Reply by GingerNinjaa** [reply]
Hi, actually I don't know how to use this library... I was trying to use sample from this repo and I give up and create question.
